algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>"(see our AQC tutorial)." -&gt; put a link to the tutorial</p><h4>"<strong>2.1 Null Space of&nbsp;H0" -&gt; contructing H0?</strong></h4><p>"bounded below by&nbsp;1/k" -&gt; bounded from below by 1/k. (maybe write each one of them in the previous sections instead here?). Don't forget to remove 'VERIFY'</p><p> </p><p>maybe explain what is the Null space (in a few words, like: in the Null space, i.e. has eigenvalue 0)</p><p> </p><p>"There exists a state&nbsp;" -&gt; maybe there exist an <strong><em>additional</em></strong> state. I  would explain that unlike the regular adiabatic algo usages, here this is not the ground state, and also there is a degeneracy in the number of states. The authors prove that if we start in the eigenstate |x&gt; and evolve slowly enough, we will remain in this eigenstate path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>